src/GitHub.Extensions/TaskExtensions.cs

```diff
@@ -20,5 +20,8 @@ public static class TaskExtensions
                 return default(T);
             }
         }
+        public static void Forget(this Task task)
```

Comment:
BTW, this is the same as https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.threading.tplextensions.forget.aspx (turns out I'm not the only one with the awesomely imaginative method naming conventions :P)